src/main/java/org/folio/rest/impl/BaseApi.java

```diff
@@ -74,7 +75,11 @@ public void addProcessingError(Error error) {
   }
 
   protected int handleProcessingError(Throwable throwable) {
-    final Throwable cause = throwable.getCause();
+    Throwable cause = throwable.getCause();
+    if(cause == null)
```

Comment:
Could you follow the project formatting style? And also please use Objects.isNull method

Reply:
fixed